### PR TITLE
#1453 Added "Talismans of Merit" quests

### PR DESCRIPTION
**Pull request by AndrewVBragg**
Added quests using whatever information could be pulled from their wowhead pages. 
See https://github.com/AeroScripts/QuestieDev/issues/1453 for links.

### CODE REVIEW

File: Database/Corrections/QuestieQuestFixes.lua [hunk 1]
```diff
@@ -230,6 +236,72 @@ function QuestieQuestFixes:Load()
             [QuestieDB.questKeys.sourceItemId] = 18746,
             [QuestieDB.questKeys.zoneOrSort] = 1637,
         },
+        [8289] = { -- #1435
+            [QuestieDB.questKeys.name] = "Talismans of Merit",
+            [QuestieDB.questKeys.startedBy] = {{14733},nil,nil},    --Quest starts and ends with Sentinel Farsong
+            [QuestieDB.questKeys.finishedBy] = {{14733},nil,},
+            [QuestieDB.questKeys.requiredLevel] = 10,               -- All info repeats, but quest has different levels for level of player
+            [QuestieDB.questKeys.questLevel] = 15,
+            [QuestieDB.questKeys.requiredRaces] = 0,                -- All races can partake in quest
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]: Actually the quests are just for Alliance

```suggestion
            [QuestieDB.questKeys.requiredRaces] = 77,                -- All races can partake in quest
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 2]
```diff
@@ -230,6 +236,72 @@ function QuestieQuestFixes:Load()
...
+            [QuestieDB.questKeys.specialFlags] = 1,                 -- Quest is repeatable
+            
+        },
+        [7921] = { -- #1435
+            [QuestieDB.questKeys.name] = "Talismans of Merit",  -- Check [8289] for comments with more information
+            [QuestieDB.questKeys.startedBy] = {{14733},nil,nil},
+            [QuestieDB.questKeys.finishedBy] = {{14733},nil,},
+            [QuestieDB.questKeys.requiredLevel] = 20,
+            [QuestieDB.questKeys.questLevel] = 25,
+            [QuestieDB.questKeys.requiredRaces] = 0,
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.requiredRaces] = 77,
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 3]
```diff
@@ -230,6 +236,72 @@ function QuestieQuestFixes:Load()
...
+            [QuestieDB.questKeys.specialFlags] = 1,
+            
+        },
+        [7887] = { -- #1435
+            [QuestieDB.questKeys.name] = "Talismans of Merit",  -- Check [8289] for comments with more information
+            [QuestieDB.questKeys.startedBy] = {{14733},nil,nil},
+            [QuestieDB.questKeys.finishedBy] = {{14733},nil,},
+            [QuestieDB.questKeys.requiredLevel] = 30,
+            [QuestieDB.questKeys.questLevel] = 35,
+            [QuestieDB.questKeys.requiredRaces] = 0,
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.requiredRaces] = 77,
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 4]
```diff
@@ -230,6 +236,72 @@ function QuestieQuestFixes:Load()
...
+            [QuestieDB.questKeys.specialFlags] = 1,
+            
+        },
+        [7888] = { -- #1435
+            [QuestieDB.questKeys.name] = "Talismans of Merit",  -- Check [8289] for comments with more information
+            [QuestieDB.questKeys.startedBy] = {{14733},nil,nil},
+            [QuestieDB.questKeys.finishedBy] = {{14733},nil,},
+            [QuestieDB.questKeys.requiredLevel] = 40,
+            [QuestieDB.questKeys.questLevel] = 45,
+            [QuestieDB.questKeys.requiredRaces] = 0,
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.requiredRaces] = 77,
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 5]
```diff
@@ -230,6 +236,72 @@ function QuestieQuestFixes:Load()
...
+            [QuestieDB.questKeys.specialFlags] = 1,
+            
+        },
+        [7886] = { -- #1435
+            [QuestieDB.questKeys.name] = "Talismans of Merit",  -- Check [8289] for comments with more information
+            [QuestieDB.questKeys.startedBy] = {{14733},nil,nil},
+            [QuestieDB.questKeys.finishedBy] = {{14733},nil,},
+            [QuestieDB.questKeys.requiredLevel] = 50,
+            [QuestieDB.questKeys.questLevel] = 55,
+            [QuestieDB.questKeys.requiredRaces] = 0,
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.requiredRaces] = 77,
```

File: Database/Corrections/QuestieQuestFixes.lua [hunk 6]
```diff
@@ -230,6 +236,72 @@ function QuestieQuestFixes:Load()
...
+            [QuestieDB.questKeys.specialFlags] = 1,
+            
+        },
+        [8289] = { -- #1435
+            [QuestieDB.questKeys.name] = "Talismans of Merit",  -- Check [8289] for comments with more information
+            [QuestieDB.questKeys.startedBy] = {{14733},nil,nil},
+            [QuestieDB.questKeys.finishedBy] = {{14733},nil,},
+            [QuestieDB.questKeys.requiredLevel] = 60,
+            [QuestieDB.questKeys.questLevel] = 60,
+            [QuestieDB.questKeys.requiredRaces] = 0,
```
Comment (BreakBB) [inline Database/Corrections/QuestieQuestFixes.lua]:
```suggestion
            [QuestieDB.questKeys.requiredRaces] = 77,
```

**Comment by BreakBB** [comment]
@AndrewVBragg Please update the race flags so we can merge this :)

**Comment by AndrewVBragg** [comment]
@BreakBB I've updated the races! Thanks for catching that.

**Comment by BreakBB** [comment]
@AndrewVBragg There is still one quest missing ;)

**Comment by AndrewVBragg** [comment]
Got it! My bad.

**Pull request merged**